src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.exception.DukeStorageException;
+import duke.task.Task;
+import duke.util.SerializeUtil;
+
+
+
+/**
+ * A storage class to handle the loading and save of the tasks.
+ */
+public class Storage {
+
+    private final String path;
+    private final String filename = "saved.duke";
```

Comment:
Hi! As this is a constant, i'm not too sure if the name should be "filename" instead of "FILE_NAME"? Although checkstyle does point it out as an error on my case but I'm not too sure.

Reply:
I think you might be right :D